tensordict/tensordict.py

```diff
@@ -959,32 +960,31 @@ def __ne__(self, other: object) -> TensorDictBase:
             tensors of the same shape as the original tensors.
 
         """
+        # avoiding circular imports
         from tensordict.prototype import is_tensorclass
 
-        if not isinstance(
-            other, (TensorDictBase, dict, float, int, torch.Tensor, MemmapTensor)
-        ) and not is_tensorclass(other):
-            return True
         if not isinstance(other, TensorDictBase) and isinstance(other, dict):
```

Comment:
move this after the TensordictBase
it will be just: `if isinstance(other, dict)`

Reply:
it needs to be before since there is no return enclosed

Reply:
but only the `isinstance(other, dict)` is necessary

Reply:
right, I overlooked it, but why you need make_tensordict? the code that checks eq/ne seems to perform actions valid also for dict.

Reply:
indeed it works without it, let's remove it then